src/style.css

```diff
@@ -1 +1,151 @@
-/* empty source */
+/* #region page */
+:root {
+  --min-width-container: 200px;
+  --content-card-margin-top: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-weight: 400;
+  font-family: "Roboto", Arial, Helvetica, sans-serif ;
+  color: #060b35;
+}
+
+/* #endregion */
+
+/* #region cardSet */
+.card {
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/221789193-63055b4d-afad-4c4a-b624-d334987c9313.png)
Create separate files for `card` and `stars` BEM blocks